<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPIP测试工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>10</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string> 连接</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>561</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipAddrText"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipPortText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pingBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ping</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tcpGrp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>类别</string>
    </property>
    <widget class="QRadioButton" name="tcpRadioBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="udpRadioBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>771</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>发送</string>
    </property>
    <widget class="QTextEdit" name="sendMsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>691</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>230</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearSendBtn">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>200</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>771</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>接受</string>
    </property>
    <widget class="QTextEdit" name="receMsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>691</width>
       <height>161</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clearReceBtn">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="toolMenu">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="toolMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
